package.json

```diff
@@ -6,26 +6,30 @@
     "  2. Go to http://localhost:8000",
     "================================================================================"
   ],
-  "name": "@aurodesignsystem/auro-breadcrumb",
+  "name": "@aurodesignsystem/auro-nav",
   "version": "0.0.0",
-  "description": "auro-breadcrumb HTML custom element",
+  "description": "auro-nav HTML custom element",
   "repository": {
     "type": "git",
-    "url": "https://github.com/AlaskaAirlines/auro-breadcrumb"
+    "url": "https://github.com/AlaskaAirlines/auro-nav"
   },
   "main": "index.js",
   "license": "Apache-2.0",
   "engines": {
     "node": ">=16.13.0"
   },
   "dependencies": {
+    "@alaskaairux/icons": "^4.19.0",
+    "@aurodesignsystem/auro-input": "^2.10.3",
+    "chalk": "^4.1.2",
     "lit-element": "^2.5.1",
-    "lit-html": "^1.4.1",
-    "chalk": "^4.1.2"
+    "lit-html": "^1.4.1"
```

Comment:
Let's build this using the latest version of Lit. You can find Lit update instructions via google.